python/nano/src/bigdl/nano/pytorch/dispatcher.py

```diff
@@ -51,7 +52,7 @@ def _get_patch_map():
     return _mapping_torch
 
 
-def patch_torch():
+def patch_torch(gpu_to_cpu: bool = False):
```

Comment:
why not default to true?

Reply:
well because it's experimental for now. But we could set it to True for finding bugs quicker :)